cypress/e2e/pages/ProbeHybridisationQC.cy.ts

```diff
@@ -0,0 +1,130 @@
+        cy.get('#labwareScanInput').type('STAN-3111{enter}');
+        selectOption('globalComment', 'Poor section quality');
+      });
+      it('populates table rows comment column automatically with the selected value(s)', () => {
+        cy.get('tbody tr').eq(0).find('td').eq(6).should('have.text', 'Poor section quality');
+      });
+    });
+  });
+  describe('Validation', () => {
+    describe('When the SCP number is not selected', () => {
+      before(() => {
+        cy.findByText('Save').click();
+      });
+      it('should display an error message requires SGP number', () => {
+        cy.findByText('SGP Number is required').should('be.visible');
+      });
+    });
+    describe('When completion time is selected in the future', () => {
+      const currentDate = new Date().toLocaleString('en-GB', { hour12: false }).split(' ');
+      before(() => {
+        cy.findByLabelText('Completion Time').clear().type('2075-01-01T10:00');
+        cy.findByText('Save').click();
+      });
+      it('should display an error message', () => {
+        cy.findByText('Please select a time on or before current time').should('be.visible');
+      });
+    });
+  });
+
+  describe('Submission', () => {
+    context('when selecting a released labware', () => {
+      before(() => {
+        cy.visit('/lab/probe_hybridisation_qc');
+        cy.msw().then(({ worker, graphql }) => {
+          worker.use(
+            graphql.mutation<RecordCompletionMutation, RecordCompletionMutationVariables>(
+              'RecordCompletion',
+              (req, res, ctx) => {
+                return res.once(
+                  ctx.errors([
+                    {
+                      message: 'The request could not be validated.',
+                      extensions: {
+                        problems: ['Labware is released: [STAN-0EB01]']
+                      }
+                    }
+                  ])
+                );
+              }
+            )
+          );
+        });
+        selectSGPNumber('SGP1008');
+        cy.get('#labwareScanInput').type('STAN-0EB01{enter}');
+        cy.findByText('Save').click();
+      });
+      it('shows an error message', () => {
+        cy.findByText('Labware is released: [STAN-0EB01]').should('be.visible');
+      });
+    });
+    context('When there is no server error', () => {
+      before(() => {
+        cy.msw().then(({ worker, graphql }) => {
+          worker.use(
+            graphql.mutation<RecordCompletionMutation, RecordCompletionMutationVariables>(
+              'RecordCompletion',
+              (req, res, ctx) => {
+                return res.once(
+                  ctx.data({
+                    recordCompletion: {
+                      operations: [
+                        {
+                          id: 10426
+                        }
+                      ]
+                    }
+                  })
+                );
+              }
+            )
+          );
+        });
```

Comment:
Please add a global mock handler for this as we do with all other operations in stan-client